src/ids/__test__/quickId.test.ts

```diff
@@ -270,3 +271,154 @@ describe("quickId", () => {
     expect(await quickId("id1,jkl", {})).toEqual(["id1", "id2"]);
   });
 });
+
+describe("quickId underscore notation", () => {
+  const dbName = "test_underscore_shorthand";
+  const db = testDbLifecycle(dbName);
+
+  beforeEach(async () => {
+    await insertDatumView({ db, datumView: idToHumanView });
+    await insertDatumView({ db, datumView: subHumanIdView });
+    await insertDatumView({ db, datumView: humanIdView });
+    await insertDatumView({ db, datumView: stateChangeView });
+    await insertDatumView({ db, datumView: timingView });
+
+    const now = new Date();
+
+    await db.put({
+      _id: "sleep:1",
+      data: { field: "sleep", value: "8h" },
+      meta: {
+        humanId: "sleep1",
+        createTime: {
+          utc: new Date(now.getTime() - 1000 * 60 * 60).toISOString(),
+        },
```

Comment:
For readability use `createTime: toDatumTime("16:00")` here. You can use the same timeStr as you would use with -t

Please also use a `setNow()` call with a time string before hand to make sure that the now reference time stays deterministic across test runs.
So `setNow("2025-05-15, 17:00")` <- use the date of writing the test for posterity to match the rest of the tests

Reply:
Look how I've used `setNow` in other places in the test to get an idea. Make sure to use `restoreNow()` in an after hook to restore it